view/src/bin/pclientd.rs

```diff
@@ -163,33 +173,43 @@ async fn main() -> Result<()> {
 
             let encoded = toml::to_string_pretty(&client_config).unwrap();
 
-            println!("{}", encoded);
-
             // Write config to directory
 
-            let mut config_dir = PathBuf::from("client_data");
-
-            config_dir.push("pclientd");
-
-            fs::create_dir_all(&config_dir)?;
-
-            let mut config_file_path = config_dir.clone();
+            let config_file_path = &mut opt.home.clone();
             config_file_path.push("config.toml");
-            let mut config_file = File::create(config_file_path)?;
+            let mut config_file = File::create(&config_file_path)?;
 
             config_file.write_all(encoded.as_bytes())?;
 
             Ok(())
         }
 
         Command::Start { host, view_port } => {
-            tracing::info!(?opt.sqlite_path, ?host, ?view_port, ?opt.node, ?opt.pd_port, "starting pclientd");
+            tracing::info!(?opt.home, ?host, ?view_port, ?opt.node, ?opt.pd_port, "starting pclientd");
 
-            let storage = penumbra_view::Storage::load(opt.sqlite_path).await?;
+            let sqlite_path = &mut opt.home.clone();
+            sqlite_path.push("pclientd-db.sqlite");
+
+            println!(
+                "Reading storage and configuration at: {:?}",
+                fs::canonicalize(&opt.home)?
+            );
+
+            let storage = penumbra_view::Storage::load(<Utf8PathBuf as AsRef<Utf8Path>>::as_ref(
+                &sqlite_path,
+            ))
+            .await?;
+
+            println!(
+                "Starting view service at node {:?} and port {:?}",
+                &opt.node, &opt.pd_port
+            );
 
             let service = ViewService::new(storage, opt.node, opt.pd_port).await?;
+            let config_file_path = &mut opt.home.clone();
+            config_file_path.push("config.toml");
 
-            let config_contents = match fs::read_to_string("./client_data/pclientd/config.toml") {
+            let config_contents = match fs::read_to_string(config_file_path) {
```

Comment:
Do we maybe want to read and validate the config data before starting the view service?